src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -539,7 +539,102 @@ void extractRecurrence(const QJsonArray &recurrence, KCalCore::Event::Ptr event,
     }
 }
 
-void jsonToKCal(const QJsonObject &json, KCalCore::Event::Ptr event, KCalCore::ICalFormat &icalFormat)
+int nearestNemoReminderStartOffset(int googleStartOffset)
+{
+    // Google supports arbitrary start offsets, whereas in Nemo's UI
+    // we only allow specific reminder offsets.
+    // See nemo-qml-plugin-calendar::NemoCalendarEvent::Reminder for
+    // those offset definitions.
```

Comment:
Nasty dependency, we should really at least allow showing custom reminder times in the calendar, but I guess we can start with rounding first.